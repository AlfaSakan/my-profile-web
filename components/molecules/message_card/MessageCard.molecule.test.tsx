import renderer from 'react-test-renderer';
import MessageCard from './MessageCard.molecule';

describe('MessageCard component test', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <MessageCard dateUnix={123456789} lastMessage="new group" name="alfa" />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
