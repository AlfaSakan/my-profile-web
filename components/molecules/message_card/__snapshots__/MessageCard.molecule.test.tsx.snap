// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageCard component test should render correctly 1`] = `
<div
  className="flex items-center justify-between p-3 cursor-pointer"
>
  <div
    className="flex"
  >
    <div
      className="rounded-full bg-gray-300 mr-4 flex items-center justify-center overflow-hidden h-10 w-10"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        style={
          Object {
            "fill": "white",
          }
        }
        viewBox="0 0 24 24"
      >
        <path
          d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
        />
      </svg>
    </div>
    <div
      className="flex flex-col"
    >
      <p>
        alfa
      </p>
      <p
        className="text-xs mt-1"
      >
        new group
      </p>
    </div>
  </div>
  <div
    className="flex flex-col justify-start self-start"
  >
    <p
      className="text-xs"
    >
      17:17
    </p>
  </div>
</div>
`;
